.github/workflows/code_quality-aarch64-darwin.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       - name: Install dependencies
         run: brew tap slp/krunkit && brew install virglrenderer clang-format llvm
 
+      - name: Create a fake init
+        run: touch init/init
```

Comment:
Why is this needed?

Reply:
To satisfy this in virtio-fs:

```
static INIT_BINARY: &[u8] = include_bytes!("../../../../../../init/init");
```

This was gated out in the EFI variant, but with the unification we need it for building both variants.